plugin/db/driver_test.go

```diff
@@ -32,16 +33,16 @@ func TestParseMigrationInfo(t *testing.T) {
 			wantErr: "",
 		},
 		{
-			filePath:         "db foo_+-=/_#?!$.__1.2.3",
+			filePath:         "db foo_+-=_#!$.__1.2.3",
```

Comment:
We are removing `/?`, is this a breaking change?

Reply:
Yeah, for example, if we get `foo/bar/foo/bar/foo/bar/db__v1__data__xx.sql`, and the template is `{{ENV_NAME}}/**/{{DB_NAME}}__{{VERSION}}__{{TYPE}}__{{DESCRIPTION}}.sql`, I think it's hard to decide who matches ENV_NAME, foo, foo/bar, foo/bar/foo or others?

Reply:
Is `{{ENV_NAME}}` allowed to contain `/` (i don't know)? If not, it seems clear `foo` is the `{{ENV_NAME}}`.

Reply:
Yeah, I means we will encounter this problem if we support '/' appear in ENV_NAME. So I banned it.

Reply:
Do we allow `{{ENV_NAME}}` to contain `/` before this PR?

Reply:
Yeah

Reply:
OK, then I think we need to confirm with existing users (we may be lucky if no one is relying on having `/` in the `{{ENV_NAME}}`), and add this breaking change to the release note after merging this PR.

Reply:
According to the offline discussion, we banned it. 👻